public/js/main.js

```diff
@@ -59,26 +59,26 @@ async function changeStatusFalse() {
   }
 }
 
-// Cafe: Delete shift
-Array.from(deleteShiftBtn).forEach(btn => {
-  btn.addEventListener('click', deleteShift)
+// Cafe: remove shift from being displayed on cafe dashboard
+Array.from(removeShiftBtn).forEach(btn => {
+  btn.addEventListener('click', removeShiftDisplay )
 })
 
-async function deleteShift() {
+async function removeShiftDisplay() {
   const shiftID = this.parentNode.parentNode.dataset.id
   try {
-    const response = await fetch('cafe/deleteShift', {
-      method: 'delete',
-      headers: { 'Content-type': 'application/json' },
+    const response = await fetch('cafe/putCafeDisplayFalse', {
+      method: 'put',

```

Comment:
This is not a usual use of a `put` request. `PUT` usually means that you are going to update the object by sending the entire object payload. What you've got here is something of an "action". I'd usually describe such an action as a POST. Consider having a look at https://restcookbook.com/